lib/src/main/scala/spinal/lib/sim/bus/wishbone/WishboneSequencer.scala

```diff
@@ -1,10 +1,26 @@
 package spinal.lib.wishbone.sim
 
+import spinal.core.log2Up
+import spinal.core.sim.simRandom
+
 import scala.collection.immutable._
 import spinal.lib.bus.misc._
+import spinal.lib.bus.wishbone.{Wishbone, WishboneConfig}
 
 object WishboneSequencer{
   def apply(builder: => Seq[WishboneTransaction]) = new WishboneSequencer(builder)
+
+  def randomGen(busConfig : WishboneConfig, upto : Int = 10) = {
```

Comment:
```suggestion
  def randomGen(busConfig : WishboneConfig, maxCnt : Int = 10) = {
```
`upto` could be for address/data/etc.